src/.nuget/Microsoft.NETCore.ILAsm/Microsoft.NETCore.ILAsm.builds

```diff
@@ -1,89 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), packaging.props))\packaging.props" />
```

Comment:
I didn't want to cause a conflict with Microsoft.TargetingPackage.Private.CoreClr, which doesn't build using these props / targets.  Also, there are enough places where we don't want to import packaging.targets, that we would have to substitute `..\..\..\dir.targets` instead, that it seemed ugly and I didn't want to do it.